src/constants.js

```diff
@@ -341,6 +341,8 @@ export const WRAPPER_DIR = 'bit_wrapper_dir';
 
 export const PACKAGE_JSON = 'package.json';
 
+export const PACKAGE_JSON_INDENT = 2; // as per NPM default
```

Comment:
PACKAGE_JSON_INDENT -> PACKAGE_JSON_DEFAULT_INDENT

Reply:
yeah, I was debating between the two. ok, I'll go with the default.